skbio/alignment/_alignment.py

```diff
@@ -627,7 +627,9 @@ def update_ids(self, ids=None, fn=None, prefix=""):
                                           "or fn is provided.")
 
         if ids is not None:
-            fn = lambda _: ids
```

Comment:
That's a really good suggestion!

On (Feb-12-15| 9:43), Jorge Ca�ardo Alastuey wrote:
> > @@ -627,7 +627,9 @@ def update_ids(self, ids=None, fn=None, prefix=""):
> >                                            "or fn is provided.")
> >  
> >          if ids is not None:
> > -            fn = lambda _: ids
> 
> :cry:
> 
> However, from the PEP8 example, maybe we can keep those super simple functions in one line
> 
> > Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier. 
> Yes:
> `def f(x): return 2*x`.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/scikit-bio/pull/819/files#r24600722